rs/messaging/src/routing/stream_builder/tests.rs

```diff
@@ -90,20 +90,16 @@ fn reject_local_request() {
 
         // With a reservation on an input queue.
         let payment = Cycles::new(100);
-        let msg = generate_message_for_test(
+        let mut msg = generate_message_for_test(
             sender,
             receiver,
             CallbackId::from(1),
```

Comment:
As far as I can tell this callback ID is either overridden later on, or, if not, we do not care what its value is. I think it might be better to remove this parameter altogether and set it internally in `generate_message_for_test` to some constant value.

Reply:
I switched this around such that it first registers and generates an ID and then uses it to generate the message.